lib/avalon/batch/entry.rb

```diff
@@ -277,15 +277,32 @@ def self.caption_language(language)
       end
       private_class_method :caption_language
 
+      def self.process_datastream(datastream, type)
+        file, label, language = ["_file", "_label", "_language"].map { |item| item.prepend(type).to_sym }
+        if datastream[file].present? && FileLocator.new(datastream[file]).exist?
```

Comment:
What do you think about changing this into an explicit return `nil`?
```suggestion
        return nil unless datastream[file].present? && FileLocator.new(datastream[file]).exist?
```